server/database/fixtures/UserSeeder.js

```diff
@@ -1,28 +1,27 @@
+const argon2 = require("argon2");
 const AbstractSeeder = require("./AbstractSeeder");
 
 class UserSeeder extends AbstractSeeder {
   constructor() {
-    // Call the constructor of the parent class (AbstractSeeder) with appropriate options
     super({ table: "user", truncate: true });
   }
 
-  // The run method - Populate the 'user' table with fake data
-
-  run() {
-    // Generate and insert fake data into the 'user' table
+  async run() {
     for (let i = 0; i < 10; i += 1) {
-      // Generate fake user data
       const fakeUser = {
-        email: this.faker.internet.email(), // Generate a fake email using faker library
-        password: this.faker.internet.password(), // Generate a fake password using faker library
-        refName: `user_${i}`, // Create a reference name for the user
+        email: this.faker.internet.email(),
+        /* eslint-disable no-await-in-loop */
+        password: await argon2.hash(this.faker.internet.password()),
```

Comment:
Juste pour info: soyez vigilant(e)s sur le fait que vus ne pourrez jamais vous connecter avec les user qui sont créé avec le seed puisque le hash des mot de passe sont fait à partir de mot de passe aléatoire (avec Faker), donc vous ne pourrez jamais retrouver le mot de passe de base.

Une solution pour contourné ça est de mettre un mot de passe en dur au lieu du "this.faker.internet.password()"